src/main/webapp/availability.js

```diff
@@ -44,7 +44,7 @@ function availabilityTableDiv() {
   return document.getElementById('table-container');
 }
 
-function updateAvailability() {
+function updateAvailability(buttonType) {
```

Comment:
Change this to a boolean indicating whether to reload the page would probably make this interface simpler.

Reply:
Done